UIKitTeamPlate/UIKitTeamPlate/AdditionalIngredientsViewController.swift

```diff
@@ -1,14 +1,180 @@
+        static let greenAlmondMilkText = "+\(Coffee.CoffeePrice.almondMilk) руб"
+        static let espressoText = "Эспрессо 50мл"
+        static let greenEspressoText = "+\(Coffee.CoffeePrice.espresso) руб"
+
+        /// Шрифт
+        static let verdanaBoldFont = "Verdana-Bold"
+    }
+
+    // MARK: - Public Properties
+
+    /// Экземпляр модели
+    var coffeeItem = Coffee()
+
+    /// Экземпляр делегата для обмена экземпляром модели между контроллерами
+    weak var delegate: TransferCoffeeOptionsDelegate?
+
+    // MARK: - Visual Components
+
+    /// Кнопка "Скрыть экран"
+    private lazy var cancelButton: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 20, y: 26, width: 14, height: 14)
+        button.setImage(UIImage(named: Constants.closeImage), for: .normal)
+        button.addTarget(self, action: #selector(closeView), for: .touchUpInside)
+        return button
+    }()
+
+    /// Заголовок "Добавить опции"
+    private lazy var addOptionLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 40, y: 53, width: 294, height: 44)
+        label.text = Constants.mainLabelText
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.font = UIFont(name: Constants.verdanaBoldFont, size: 18)
+        return label
+    }()
+
+    /// Текстовые лейблы для опций
+    private lazy var standardMilkLabel = addLabel(
+        yPoint: 124,
+        blackText: Constants.milkText,
+        greenText: Constants.greenMilkText
+    )
+    private lazy var syrupLabel = addLabel(
+        yPoint: 174,
+        blackText: Constants.syrupText,
+        greenText: Constants.greenSyrupText
+    )
+    private lazy var soyaMilkLabel = addLabel(
+        yPoint: 224,
+        blackText: Constants.soyaMilkText,
+        greenText: Constants.greenSoyaMilkText
+    )
+    private lazy var almondMilkLabel = addLabel(
+        yPoint: 274,
+        blackText: Constants.almondMilkText,
+        greenText: Constants.greenAlmondMilkText
+    )
+    private lazy var espressoLabel = addLabel(
+        yPoint: 324,
+        blackText: Constants.espressoText,
+        greenText: Constants.greenEspressoText
+    )
+
+    /// Свитчеры для выбора опций
+    private lazy var milkSwitch = UISwitch(frame: CGRect(x: 301, y: 124, width: 31, height: 31))
+    private lazy var syrupSwitch = UISwitch(frame: CGRect(x: 301, y: 174, width: 31, height: 31))
+    private lazy var soyaMilkSwitch = UISwitch(frame: CGRect(x: 301, y: 224, width: 31, height: 31))
+    private lazy var almondMilkSwitch = UISwitch(frame: CGRect(x: 301, y: 274, width: 31, height: 31))
+    private lazy var espressoSwitch = UISwitch(frame: CGRect(x: 301, y: 324, width: 31, height: 31))
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+
+    /// Установка вью
```

Comment:
```suggestion
    /// Установка вью
```
Тоже все коменты над методами убираем